py/selenium/webdriver/chromium/service.py

```diff
@@ -40,10 +40,12 @@ def __init__(
         env: typing.Optional[typing.Mapping[str, str]] = None,
         **kwargs,
     ) -> None:
-        self.service_args = service_args or []
```

Comment:
just curious, is this not a good convention?

Reply:
It is good one-liner assignment, kind of short-cut. But most of the times I see explicit check for truthiness checking against `None`.  I think both ways are fine.